roles/transmissionx/tasks/template.yml

```diff
@@ -43,6 +43,7 @@
       - "/mnt:/mnt"
       - "/opt/transmission{{ rolename }}:/opt/transmission{{ rolename }}"
       - "/mnt/local/downloads/torrents/transmission{{ rolename }}:/data"
+      - "/mnt/local/downloads/torrents/transmission{{ rolename }}/watched:/watch"
```

Comment:
The watched folder cannot be changed: https://github.com/linuxserver/docker-transmission/issues/100